AirbnbSwiftFormatTool/AirbnbSwiftFormatTool.swift

```diff
@@ -0,0 +1,84 @@
+import ArgumentParser
+import Foundation
+
+/// A command line tool that formats the given directories using SwiftFormat and SwiftLint,
+/// based on the Airbnb Swift Style Guide
+@main
+struct AirbnbSwiftFormatTool: ParsableCommand {
+
+  // MARK: Internal
+
+  @Argument(help: "The directories to format")
+  var directories: [String]
+
+  @Option(help: "The abosolute path to a SwiftFormat binary")
+  var swiftFormatPath: String
+
+  @Option(help: "The path to use for SwiftFormat's cache")
+  var swiftFormatCachePath: String?
+
+  @Option(help: "The abosolute path to a SwiftLint binary")
+  var swiftLintPath: String
+
+  @Option(help: "The path to use for SwiftLint's cache")
+  var swiftLintCachePath: String?
+
+  @Flag(help: "When true, source files are not reformatted")
+  var lint = false
+
+  @Option(help: "The absolute path to the SwiftFormat config file")
+  var swiftFormatConfig = Bundle.module.path(forResource: "airbnb", ofType: "swiftformat")!
+
+  @Option(help: "The absolute path to the SwiftLint config file")
+  var swiftLintConfig = Bundle.module.path(forResource: "swiftlint", ofType: "yml")!
+
+  mutating func run() throws {
+    try swiftFormat.run()
```

Comment:
I wondered if instead wanted to link to the code for SwiftFormat and SwiftLint but executing the binaries is probably more robust.